packages/modals/src/styled/StyledDrawerModalHeader.ts

```diff
@@ -16,6 +17,11 @@ export const StyledDrawerModalHeader = styled(StyledHeader).attrs({
   'data-garden-version': PACKAGE_VERSION
 })`
   padding: ${props => props.theme.space.base * 5}px;
+  ${props =>
+    props.isCloseButtonPresent &&
+    `padding-${props.theme.rtl ? 'left' : 'right'}: ${
+      props.theme.space.base * (baseMultipliers.size + baseMultipliers.side)
```

Comment:
This should be a base-4 value, precisely the same as `Sheet`. See https://github.com/zendeskgarden/react-components/blob/028957b8a4f0f9fa63ea09fbeb3a59549381d15e/packages/chrome/src/styled/sheet/StyledSheetHeader.ts#L31.